packages/jest-preset-mc-app/fail-tests-because-there-was-an-unhandled-rejection.lock

```diff
@@ -0,0 +1 @@
+undefined
```

Comment:
This file should be ignored

Reply:
Good point. Amended into the last commit: ab81f42